Assets/Mirror/Runtime/LocalConnections.cs

```diff
@@ -72,13 +77,25 @@ internal override bool Send(ArraySegment<byte> segment, int channelId = Channels
 
         internal void Update()
         {
+            if (connectionToClient == null) { return; }
```

Comment:
You need to call writer.SetLength(0) at the end of this method,
otherwise the writer will just grow forever until the app crashes.